### ISSUE TITLE
EpicLoot Crafting Material Icons issue

### ISSUE THREAD

**Issue by SacrificialGoatTwin** [RandyKnapp/ValheimMods#696]
Hey :D

I have changed the colour codes for the item drops to match Diablos colours sortof
I changed the crafting materials colours too. In the inventory the icons I changed are correct
However in the Enchanting table, only the 1st resource of the 3 required for enchanting matches. the other 2 are the original icons

![image](https://github.com/user-attachments/assets/40165853-b95f-4f80-808f-a4fd5b1627f2)

**Comment by SacrificialGoatTwin** [comment]
![image](https://github.com/user-attachments/assets/16dd91c7-c8b8-4033-8ca6-f180497b4e82)


Couple of images as an example

**Comment by SacrificialGoatTwin** [comment]
Ok, looks like when I updated them from my client, the server itself did not reflect those changes.
I realised this when my friend logged in and said his were default in the chest too.
I have manually updated the ones on the server and it has resolved it.

The reason why the first one had the correct colour was because the colour was determined by its background, not the icon like the others..... my bad
